server/planning/utils.py

```diff
@@ -1,10 +1,97 @@
-from typing import Union
+from typing import Union, List, Dict, Any, TypedDict
 from bson.objectid import ObjectId
 from bson.errors import InvalidId
+from datetime import datetime
+from flask_babel import format_time, format_datetime, lazy_gettext
+from eve.utils import str_to_date
+import arrow
+from flask import current_app as app
+
+
+class FormattedContact(TypedDict):
+    name: str
+    organisation: str
+    email: List[str]
+    phone: List[str]
+    mobile: List[str]
+    website: str
+
+
+ALL_DAY_SECONDS = 89999  # Number of seconds for an all-day event
+MULTI_DAY_SECONDS = 90000  # Number of seconds for an multi-day event
```

Comment:
it could be like:
```py
MULTI_DAY_SECONDS = 24 * 60 * 60
ALL_DAY_SECONDS = MULTI_DAY_SECONDS - 1
```
otherwise I'm not really sure where are those numbers coming from